daffodil-cli/src/it/scala/org/apache/daffodil/unparsing/TestCLIUnparsing.scala

```diff
@@ -332,4 +332,13 @@ class TestCLIunparsing {
     } (ExitCode.Success)
   }
 
+  @Test def test_XXX_CLI_Unparsing_Stream_sax(): Unit = {
+    val schema = path("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/cli_schema_02.dfdl.xsd")
+    val input = path("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/input19.txt")
+
+    runCLI(args"unparse -I sax --stream -s $schema $input") { cli =>
+      cli.expect("123")
+    } (ExitCode.Success)
```

Comment:
This was to get additional code coverage. Because of the special way that SAX works in the CLI, the only way to unparse an `Array[Byte]` is with the `--stream` option. This is because the normal unparse CLI subcommands will use an InputStream, and performance will use the new `SAXEvent` stuff.